test/system_test.py

```diff
@@ -408,9 +408,9 @@ def test_install_nwp_test_cpu_cce(self):
     @pytest.mark.no_balfrin  # config file does not exist for this machine
     def test_install_exclaim_test_gpu_dsl(self):
         spack_env_dev_install_and_test(
-            'config/cscs/spack/v0.18.1.7/daint_dsl_nvhpc',
-            'git@github.com:C2SM/icon.git',
-            'ci_dsl',
+            'config/cscs/spack/v0.20.1.2/daint_dsl_nvhpc',
+            'git@github.com:C2SM/icon-exclaim.git',
+            'new_tag',  # will change back after new tag is released
```

Comment:
Will this be the icon-exclaim testcase in spack-c2sm?

If so it needs to be a tag, not a branch, see #854

Reply:
So my plan was to change `new_tag` to a tag once the latest tag of `icon-dsl` is released. But this latest tag will depend on the upcoming spack-c2sm tag for `v0.20.1.2`.  Kind of a circular dependency, if we require the next spack-c2sm tag to not have `new_tag` in the system_test.py.. 

Not entirely sure how to proceed.